test/clj/rems/api/test_categories.clj

```diff
@@ -21,66 +22,66 @@
                                                      :en "integration test"}
                               :category/children []}]
 
-    (testing "fetch nonexistent"
-      (let [response (api-response :get "/api/categories/9999999" nil
-                                   +test-api-key+ owner)]
-        (is (response-is-not-found? response))
-        (is (= {:error "not found"} (read-body response)))))
-
-    (testing "create"
-      (let [category (api-call :post "/api/categories"
-                               create-category-data
-                               +test-api-key+ owner)]
-        (is (:success category))
-        (is (int? (:category/id category)))
-
-        (testing "and fetch"
-          (let [result (api-call :get (str "/api/categories/" (:category/id category)) nil
-                                 +test-api-key+ owner)
-                expected (merge create-category-data
-                                {:category/id (:category/id category)
-                                 :category/display-order 1000000})]
-            (is (= expected result))))))
-
-    (testing "adding category as children"
-      (let [owner "owner"
-            dep-category (api-call :post "/api/categories"
-                                   create-category-data
-                                   +test-api-key+ owner)]
-        (is (:success dep-category))
-        (is (int? (:category/id dep-category)))
+    (doseq [user-id [owner org-owner]]
```

Comment:
the diff looks nasty, but basically every category test is run twice, once for each role `[owner org-owner]`

Reply:
The hide whitespace option works 👍